mw-webapp/src/component/header/Header.module.scss

```diff
@@ -5,13 +5,12 @@
   justify-content: space-between;
   align-items: center;
   width: 100%;
-  padding: $header-padding;
-  background: $header-background;
+  padding: 0 1rem;
+  background: rgb(85, 122, 149);
 }
 
 .title {
   display: flex;
-  padding-top: $title-padding;
```

Comment:
No no no, title is used in many files, just pass class name from props into title and set padding 0 in it